.github/workflows/e2e-tests.yml

```diff
@@ -63,6 +65,13 @@ jobs:
       matrix:
         service: ${{ fromJson(needs.service-matrix.outputs.services) }}
     steps:
+      - name: Debug Run or Skip Errors
```

Comment:
Run of Skip was evaluating to False and the e2e tests did not run.
I added these echoes to debug and then the tests ran. 🤷‍♂️ 
I suggest we leave the echoes for future debugging.

Reply:
The tests ran because you've now modified this file, which is included in the "common" set, and therefore triggers "run-all". So the test for "account" modification may still be broken in some way.